src/tailwind.css

```
@@ -1,3 +1,2 @@
-@tailwind base;
-@tailwind components;
-@tailwind utilities;
+@import 'tailwindcss';
+@plugin "tailwindcss-primeui";
```

```suggestion
@custom-variant dark (&:where(.app-dark, .app-dark *));
```

Based on tailwind v4 you may need this: https://tailwindcss.com/docs/dark-mode#toggling-dark-mode-manually